takoyaki/src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +14,18 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+  public final static class kHooper {
```

Comment:
nit: spelling: hopper instead of hooper

```suggestion
  public final static class kHopper {
```